src/relayFeeCalculator/chain-queries/baseQuery.ts

```diff
@@ -146,8 +167,9 @@ export class QueryBase implements QueryInterface {
         gasLimit: nativeGasCost, // prevents additional gas estimation call
       });
       const l1GasCost = await (provider as L2Provider<providers.Provider>).estimateL1GasCost(populatedTransaction);
+      const scaledL1GasCost = l1GasCost.mul(opStackL1GasCostMultiplier).div(fixedPointAdjustment);
```

Comment:
Does this mean we scale twice for OP stack chains? Once in `getGasPriceEstimate` and once here?

Reply:
I don't believe so but it could be good to have @nicholaspai confirm - I will defer the approval to this answer

Reply:
we won't scale twice. The l1 gas cost is being scaled independently of the base fee or priotiy fee, which is already computed on line 156